app/assets/javascripts/sequence_show.js

```diff
@@ -512,3 +535,33 @@ function init_sequence_show(data) {
         }
     }
 }
+
+function toggle_column(col) {
```

Comment:
Yes, but you can't simply move them. You must get rid of the inline JavaScript first.


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3475) by @bmesuere on Mon Nov 09 2015 at 22:47._